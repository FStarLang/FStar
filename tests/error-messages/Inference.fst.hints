[
  "ðh…9€\u007fqÅÙ0w2a^&ù",
  [
    [
      "Inference.alloc",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query",
        "equation_FStar.HyperStack.ST.m_rref",
        "equation_FStar.HyperStack.ST.mref",
        "equation_FStar.Monotonic.DependentMap.imap",
        "equation_FStar.Monotonic.DependentMap.t",
        "refinement_interpretation_Tm_refine_1a1278f1aecd37594ba20d888b7fd230",
        "refinement_interpretation_Tm_refine_6fbd4d82bb3e7e5b2939e88df41fcbe3",
        "refinement_interpretation_Tm_refine_8e1a63f5f0eda9c61cb76c338f52ad38",
        "refinement_interpretation_Tm_refine_d17e5cc72481ba65d7fc920fac08f000",
        "refinement_interpretation_Tm_refine_f596a1e23d65ebe54b463345ece4e00f",
        "typing_FStar.Monotonic.DependentMap.empty"
      ],
      0,
      "13c919eb742075922ed8deefe7f8bce0"
    ]
  ]
]