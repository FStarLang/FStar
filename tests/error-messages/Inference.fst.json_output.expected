>> Got issues: [
{"msg":["Subtyping check failed","Expected type Prims.eqtype\ngot type Type0","The SMT solver could not prove the query. Use --query_stats for more details."],"level":"Error","range":{"def":{"file_name":"Prims.fst","start_pos":{"line":81,"col":23},"end_pos":{"line":81,"col":30}},"use":{"file_name":"Inference.fst","start_pos":{"line":20,"col":14},"end_pos":{"line":20,"col":15}}},"number":19,"ctx":["While checking for top-level effects","While typechecking the top-level declaration `let alloc_fails`","While typechecking the top-level declaration `[@@expect_failure] let alloc_fails`"]}
{"msg":["Subtyping check failed","Expected type Prims.eqtype\ngot type Type0","The SMT solver could not prove the query. Use --query_stats for more details."],"level":"Error","range":{"def":{"file_name":"Prims.fst","start_pos":{"line":81,"col":23},"end_pos":{"line":81,"col":30}},"use":{"file_name":"Inference.fst","start_pos":{"line":20,"col":14},"end_pos":{"line":20,"col":15}}},"number":19,"ctx":["While typechecking the top-level declaration `let alloc_fails`","While typechecking the top-level declaration `[@@expect_failure] let alloc_fails`"]}
>>]
